taxonomic_dist_by_soil_layer/R/taxonomic_dist_soil_layer_R.ipynb

```diff
@@ -14,85 +14,21 @@
    "cell_type": "code",
```

Comment:
<p>Text rendering oddly in previous chunk</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>